### DISCUSSION TITLE
TEA-DT Natural Environment Assurance Cases

### DISCUSSION THREAD

**Discussion by chrisdburr** [alan-turing-institute/AssurancePlatform#503] [General]
Please add your assurance cases to this conversation by Tuesday 25th June (End of Day). You can upload the json file by clicking on the 'Paste, drop, or click to add files' button below the text box.

Please name your file with the name of your organisation, so that other participants know which one to vote for (e.g. `alan-turing-institute.json`).

You can then vote for another project team's assurance case by using this form: https://forms.office.com/Pages/ResponsePage.aspx?id=p_SVQ1XklU-Knx-672OE-dxav9xbraJGpD_BP5Yb3HhUREZZNFFRQk4zOE5IMEZFOFk5VDhHWDVBSC4u

All votes need to be cast by Friday 28th June. Please note that any votes submitted after this date or for your own project will not be counted.

Winners will be notified on Monday 1st July.

**Comment by JimCircadian** [comment]
Basic assurance case for sea-ice predictions underlying real-world applications. Worked on during the second workshop, needs a lot more work to be comprenhensive / useful, but illustrative of the potential and has definitely whet my appetite!

[british-antarctic-survey.json](https://github.com/user-attachments/files/16015296/british-antarctic-survey.json)

**Comment by Peathugger** [comment]
Hi all, here's the second iteration of the assurance case for AI4Peat. Really good template, generated some useful conversations and highlighted the gaps. Started to use the notes section for gaps and thoughts, as well as property claims for known actions we need to perform towards improved assurance. Definitely lots more to review and apply! thank you.

[Accuracy (TEA-DT Natural Environment Workshop)-2024-5-27T16-33-57.json](https://github.com/user-attachments/files/16017660/Accuracy.TEA-DT.Natural.Environment.Workshop.-2024-5-27T16-33-57.json)

**Comment by MichaelMcGlade** [comment]
[Michael McGlade SPLASH Project.json](https://github.com/user-attachments/files/16021977/Michael.McGlade.SPLASH.Project.json)
